CMakeLists.txt

```
@@ -122,7 +122,7 @@ if(_use_onemkl_interfaces)
         FetchContent_Declare(
                 onemkl_interfaces_library
```

Overall, it would be nice to rename all use cases of `onemkl_interfaces` to `onemath` in a future PR.
```suggestion
                onemath_library
```

---

Thank you! Renamed.